### DISCUSSION TITLE
Multiple pool controllers

### DISCUSSION THREAD

**Discussion by cutfuse** [tagyoureit/nodejs-poolController#456] [App and O/S Config Questions]
I've been using an easytouch with physical rs485 adapter to the the nodejs pool controller on our hot tub for a while...now we've added another easytouch for our pool which is quite far away, so I was going to use a rs485 to Ethernet adapter over fiber to connect back to the house. Does this server support having multiple configurations running on the same server for the 2 separate easytouch systems?

**Comment by rstrouse** [comment]
Unfortunately, the EasyTouch controllers make that pretty difficult.  You could however have two separate controllers and mirror the functions on each of them.  It would be a bit complicated to set up but the gist would be that, one instance of njsPC would talk to the other over websockets.

**Reply by cutfuse** [reply]
Thanks for the response, it's a good point, I think running 2 instances of the pool controller would work fine...the only interface is homebridge to HomeKit. It may be a more relevant question for the homebridge poolcontroller plugin

**Comment by tagyoureit** [comment]
Intellitouch and Intellicenter "D" models (dual body) supports separate pool and spa bodies out of the box, and RS485 can be run a long distance.  Just curious why you went with a separate easytouch when you could have upgraded and only had a single controller for both?

**Reply by cutfuse** [reply]
Well the easytouch supports multiple bodies too, but in this case they are 2 completely separate systems in 2 locations a few hundred feet apart. The spa is  adjacent to the house, but the pool is further away. I know rs485 has a longer range than Ethernet, but I've gotten out of the habit of using copper underground because I've lost a lot of gear to lighting strikes over the years, even with surge protectors and drain wires. If the rs485 to Ethernet adapter doesn't work then I'll pull a 4 wire back to the house. I'm also heavily virtualized so physical adapters aren't really ideal. If the adapter works I'll probably switch the spa over to that method too. Thanks for your feedback and work on this project, it's amazing. So much better than the ScreenLogic garbage.

**Reply by rstrouse** [reply]
RS485 Will do more than a mile without repeaters.  There is however no transient voltage circuit that can absorb the power of Thor.   If you have the skills you can join two njsPC instances with sockets and custom bindings.
